test/project1/corectl-alt.yml

```diff
@@ -0,0 +1,17 @@
+engine: localhost:9076
```

Comment:
Do we really need additional project files? Kind of confusing to have multiple configs in the same folder. Can we move the test case that requires this config to another project instead?